tests/applications/Applications.DicomLoader.Tests/Applications.DicomLoader.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Same here with `/Directory.Build.props` and `/tests/Directory.Build.props`

Reply:
Both .csproj files thinned out now to avoid duplication.